src/winetricks

```diff
@@ -532,8 +532,11 @@ w_try_regedit32()
 {
     # on windows, doesn't work without cmd /c
     case "$W_PLATFORM" in
-        windows_cmd|wine_cmd) cmdc="cmd /c";;
-        *) unset cmdc ;;
+        windows_cygwin|wine_cmd) cmdc="cmd /c" ;;
+        nix_linux|nix_freebsd|nix_darwin) unset cmdc ;;
+        nix_redox) w_die "Platform '$W_PLATFORM' is not implemented in w_try_regedit32()" ;;
```

Comment:
If a platform legitimately doesn't have regedit, it's completely broken and there's no reason for adding 'support' for it.

Reply:
Agree, but assuming that i'm rewriting winetricks_get_platform and that i was using `windows_windows` for testing i implemented it too assuming it being efficient on resources and my general ethicacy is to support everything.